source/lib/JMSLab/autofill.py

```diff
@@ -10,15 +10,15 @@ def Autofill(var, format = "{}", namespace = None):
             parent = parent.f_back
         
         if var not in parent.f_locals.keys() or parent is None:
-            raise Warning(f"Autofill: Variable '{var}' not found")
+            raise Exception(f"Autofill: Variable '{var}' not found")
         
         namespace = parent.f_locals
     
     commandname, content = var, namespace[var]
     
     return newcommand.format(commandname, content)
 
-def GenerateAutofillMacros(autofill_lists,  autofill_formats, autofill_outfile):
+def GenerateAutofillMacros(autofill_lists,  autofill_outfile, autofill_formats = "{:.2f}"):
     """
```

Comment:
Minor: could we use `'''` for consistency in style with other scripts? Eg, see [log.py](https://github.com/JMSLab/Template/blob/db8ac17cbb45a65ec967d7bfdf5af94afc0c6329/source/lib/JMSLab/log.py)

Reply:
@santiagohermo do you mean to replace all the "" with ''? I still see some discrepancies in log.py for instance:
https://github.com/JMSLab/Template/blob/db8ac17cbb45a65ec967d7bfdf5af94afc0c6329/source/lib/JMSLab/log.py#L17

vs

https://github.com/JMSLab/Template/blob/db8ac17cbb45a65ec967d7bfdf5af94afc0c6329/source/lib/JMSLab/log.py#L23

Do let me know if I misunderstood, thanks!

Reply:
Thanks @jinfu93! I see that we are not consistent with using `'` or `"` throughout. In [the comment](https://github.com/JMSLab/Template/pull/57#discussion_r851325117) I just meant to use `'''` instead of `"""` to open and close the comment that explained the function, ie, [this one](https://github.com/JMSLab/Template/blob/db8ac17cbb45a65ec967d7bfdf5af94afc0c6329/source/lib/JMSLab/autofill.py#L22-L34)

Reply:
Got it thanks for clarifying! Change made in [6ab726e](https://github.com/JMSLab/Template/pull/57/commits/6ab726e3e1f3fd710092d68c81ff73b9e6d2019d).